app/code/Magento/WishlistGraphQl/Model/Resolver/UpdateProductsInWishlist.php

```diff
@@ -160,4 +185,110 @@ private function getWishlist(?int $wishlistId, ?int $customerId): Wishlist
 
         return $wishlist;
     }
+
+    /**
+     * Update wishlist Item and set data from request
+     *
+     * @param int|Item $itemId
+     * @param DataObject $buyRequest
+     * @param wishlistFactory $wishlist
+     * @param null|array|DataObject $params
+     *
+     * @return $this
+     */
+    public function updateItem($itemId, $buyRequest,$wishlist,$params = null)
+    {
+        $item = null;
+        if ($itemId instanceof Item) {
+            $item = $itemId;
+            $itemId = $item->getId();
+        } else {
+            $item = $wishlist->getItem((int)$itemId);
+        }
+        if (!$item) {
+            throw new GraphQlInputException(__('We can\'t specify a wish list item.'));
+        }
+
+        try {
+            $product = $item->getProduct();
+            $productId = $product->getId();
+            if ($productId) {
+                if (!$params) {
+                    $params = new DataObject();
+                } elseif (is_array($params)) {
+                    $params = new DataObject($params);
+                }
+                $params->setCurrentConfig($item->getBuyRequest());
+                $buyRequest = $this->catalogProduct->addParamsToBuyRequest($buyRequest, $params);
+                $buyRequest->setData('action', 'updateItem');
+                $product->setWishlistStoreId($item->getStoreId());
+                $cartCandidates = $product->getTypeInstance()->processConfiguration($buyRequest, clone $product);
+                /**
+                 * Error message
+                 */
+                if (is_string($cartCandidates)) {
+                    return $cartCandidates;
+                }
+
+                /**
+                 * If prepare process return one object
+                 */
+                if (!is_array($cartCandidates)) {
+                    $cartCandidates = [$cartCandidates];
+                }
+
+                foreach ($cartCandidates as $candidate) {
+                    if ($candidate->getParentProductId()) {
+                        continue;
+                    }
+                    $candidate->setWishlistStoreId($item->getStoreId());
+
+                    $qty = $buyRequest->getData('qty') ? $buyRequest->getData('qty') : 1;
+                    $item->setOptions($candidate->getCustomOptions());
+                    $item->setQty($qty)->save();
+                }
+                $wishlist->save();
```

Comment:
Please, do not use entity model for saving (deprecated approach). Use repo or resource model instead.